crt_portal/static/js/form_letter.js

```diff
@@ -77,6 +74,9 @@
     }
   };
 
+  let cancel_modal = document.getElementById('intake_template_cancel');
+  root.CRT.cancelModal(modal, cancel_modal, null, reset);
```

Comment:
```suggestion
  const cancelModal = document.getElementById('intake_template_cancel');
  root.CRT.cancelModal(modal, cancelModal, null, reset);
```

default to const (I don't think this gets changed)

plus camelCase for Javascript (I know some of the existing code doesn't use camelCase, but new code or simple modifications can)